AsyncNats/Util/NatsPublishChannel.cs

```diff
@@ -0,0 +1,122 @@
+﻿namespace EightyDecibel.AsyncNats
+{
+    using EightyDecibel.AsyncNats.Messages;
+    using System;
+    using System.Collections.Concurrent;
+    using System.Threading;
+    using System.Threading.Channels;
+    using System.Threading.Tasks;
+
+
+    public delegate void SerializeDelegate(Span<byte> writeBuffer);
+
+    internal class NatsPublishChannel
+    {
+        public int DefaultBufferLength { get; set; } = 1024 * 64;
+
+        readonly struct NatsPublishAction
```

Comment:
This structure isn't used?

Reply:
Right. Leftover from some other approach.